lldb/test/API/lang/swift/embedded/nested_frame_variable/main.swift

```diff
@@ -0,0 +1,26 @@
+struct S {
+  func s1() {
+    struct S2 {
+      func s2() {
+        func s2_2() {
+          struct S3 {
```

Comment:
does this get noticeable different if, e.g., S3 were a class?

Reply:
Ok I changed this to use classes, enums and structs, plus a private discriminator